### PR TITLE
APIGOV-22891 Remove "scopes" field in ARD for oauth resources with a new migration

### PR DESCRIPTION
**Pull request by tsjohns9**
- New migration to remove scopes from any ARD

### CODE REVIEW

File: pkg/apic/servicebody.go [hunk 1]
```diff
@@ -96,37 +94,24 @@ func (s *ServiceBody) GetCredentialRequestDefinitions() []string {
...
-						provisioning.NewSchemaPropertyBuilder().
-							SetName("scopes").
-							SetLabel("Scopes").
-							IsArray().
-							AddItem(
-								provisioning.NewSchemaPropertyBuilder().
-									SetName("scope").
-									IsString().
-									SetEnumValues(oauthScopes)))).Register()
+		_, err := provisioning.NewAccessRequestBuilder(s.setAccessRequestDefinition).Register()
```
Comment (tsjohns9) [inline pkg/apic/servicebody.go:114]: Still creating the ARD here, just not adding any scope stuff.

Comment (sbolosan) [reply pkg/apic/servicebody.go:114]: should I basically follow this as part of the marketplacemigration.go (func registerAccessRequestDefinition) as well? Meaning remove the scopes?

**Review by dfeldick — APPROVED** [review]

File: pkg/apic/servicebody.go (hunk 1 above)
Comment (tsjohns9) [reply pkg/apic/servicebody.go:114]: Scopes need to be removed wherever they are being added. If there is another spot I missed I can remove it, just let me know.

Comment (sbolosan) [reply pkg/apic/servicebody.go:114]: Ill update it in my branch.  Working on an issue with mp migration. So easy enough for me to update mine

**Review by sbolosan — APPROVED** [review]

File: pkg/apic/servicebody.go (hunk 1 above)
Comment (sbolosan) [reply pkg/apic/servicebody.go:114]: I have it updated in my branch and it is working, so we good. I approved.

**Review by sbolosan — DISMISSED** [review]

**Review by dfeldick — APPROVED** [review]

**Review by dfeldick — APPROVED** [review]

**Review by sbolosan — APPROVED** [review]

**Review by sbolosan — APPROVED** [review]

**Review by dfeldick — APPROVED** [review]

**Pull request merged**